src/components/LobbyScreen.vue

```diff
@@ -19,20 +19,28 @@
   -->
 
 <template>
-	<div class="lobby emptycontent">
-		<div class="icon icon-lobby" />
-		<h2>{{ currentConversationName }}</h2>
-		<p>{{ message }}</p>
+	<div class="lobby">
+		<div class="lobby emptycontent">
+			<div class="icon icon-lobby" />
+			<h2>{{ currentConversationName }}</h2>
+			<p>{{ message }}</p>
+		</div>
+		<SetGuestUsername />
```

Comment:
Missing check for guest?
Also normal users end up in the lobby

Reply:
Check for guest is in the parent component!

Reply:
feature/noid/show-selected-participants-in-new-group-creation

Reply:
Sorry you're right :)